.github/ci/presubmit.sh

```diff
@@ -5,6 +5,9 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# DO NOT SUBMIT
+curl --version
+
```

Comment:
Thanks for the review, although feel free to wait until I add someone as reviewer - this is in here to test the presubmit and will be removed when the change is ready. I'll also try to add WIP in future to avoid wasted review cycles.